components/alertbanner/index.css

```diff
@@ -9,81 +9,90 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "./themes/express.css";
-
 .spectrum-AlertBanner {
   --spectrum-alert-banner-min-height: var(--spectrum-alert-banner-minimum-height);
-  --spectrum-alert-banner-size: var(--spectrum-alert-banner-width);
+  --spectrum-alert-banner-inline-size: var(--spectrum-alert-banner-width);
   --spectrum-alert-banner-font-size: var(--spectrum-font-size-100);
+
+  /* @todo replace with font-size specific line-height token when one is available. results in a line-height of 18px with font-size-100. */
+  --spectrum-alert-banner-line-height: 1.28571428; /* stylelint-disable number-max-precision */
```

Comment:
There is no line-height defined for this component in the specs currently. Not setting one was resulting in a different line-height between Chrome and Firefox because they have different defaults. This is to match the 18px measured on the design.

Reply:
This is now defined on the spec and is changed to use `--spectrum-line-height-100`.